soc/arm/nordic_nrf/nrf52/Kconfig.soc

```diff
@@ -44,6 +44,11 @@ config SOC_NRF52810
 
 config SOC_NRF52832
 	depends on SOC_SERIES_NRF52X
+	select NRF_HW_TIMER0
```

Comment:
Why not for nRF52810, too?

Reply:
could you confirm? I would like to merge this PR to topic-counters branch.